components/signup/signup_ldap/signup_ldap.jsx

```diff
@@ -6,15 +6,19 @@ import React from 'react';
 import {FormattedHTMLMessage, FormattedMessage} from 'react-intl';
 import {Link} from 'react-router-dom';
 
-import {browserHistory} from 'utils/browser_history';
 import {trackEvent} from 'actions/diagnostics_actions.jsx';
 import * as GlobalActions from 'actions/global_actions.jsx';
 import {addUserToTeamFromInvite} from 'actions/team_actions.jsx';
 import {loadMe, webLoginByLdap} from 'actions/user_actions.jsx';
+
 import * as Utils from 'utils/utils.jsx';
+import {browserHistory} from 'utils/browser_history';
+
 import logoImage from 'images/logo.png';
+
 import BackButton from 'components/common/back_button.jsx';
 import FormError from 'components/form_error.jsx';
+import SiteNameAndDescription from 'components/common/site_name_and_description.jsx';
```

Comment:
nit: Consider importing as just `components/common/site_name_and_description`? Then, if that component gets migrated to a connected component and we introduce a directory at that path, with an `index.jsx` therein, everything "just works" without needing to change all the references.